library/alloc/src/string.rs

```diff
@@ -1059,7 +1059,7 @@ impl String {
     #[inline]
     #[must_use = "`self` will be dropped if the result is not used"]
     #[stable(feature = "rust1", since = "1.0.0")]
-    #[rustc_const_unstable(feature = "const_vec_string_slice", issue = "129041")]
+    #[rustc_const_stable(feature = "const_vec_string_slice", since = "CURRENT_RUSTC_VERSION")]
     pub const fn into_bytes(self) -> Vec<u8> {
```

Comment:
This one will need a `#[rustc_allow_const_fn_unstable(const_precise_live_drops)]`